services/rfq/api/rest/server.go

```diff
@@ -547,20 +548,33 @@ func (r *QuoterAPIServer) PutRFQRequest(c *gin.Context) {
 		span.SetAttributes(attribute.String("passive_quote_dest_amount", *passiveQuote.DestAmount))
 	}
 	quote := getBestQuote(activeQuote, passiveQuote)
+	quoteType := quoteTypeActive
+	if activeQuote == nil {
+		quoteType = quoteTypePassive
+	}
```

Comment:
This assumes that the active quote would always be the result of `getBestQuote`, which is a very big assumption. We should derive the quote type at the same time we choose the best quote imo.

Reply:
Good idea- we use `getBestQuote()` in several places, so instead of modifying that func I think this would work:

```
	var quoteType string
	if quote == activeQuote {
		quoteType = quoteTypeActive
	} else if quote == passiveQuote {
		quoteType = quoteTypePassive
	}
```